app/src/main/java/net/squanchy/schedule/view/ScheduleViewPagerAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package net.squanchy.schedule.view
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.novoda.viewpageradapter.ViewPagerAdapter
+import net.squanchy.R
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.schedule.domain.view.SchedulePage
+import java.util.*
+
+class ScheduleViewPagerAdapter(private val context: Context) : ViewPagerAdapter<ScheduleDayPageView>() {
+
+    private var pages = emptyList<SchedulePage>()
+
+    private var listener: OnEventClickedListener? = null
```

Comment:
This is never null, passed in as non-null. Should be lateinit maybe

Reply:
479360f48d2416fbe3115362dc066713df8b8385